src/App.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import { BrowserRouter, Redirect, Route, Switch } from 'react-router-dom';
 import { LoginScreen } from './modules/LoginScreen';
 
 export const App: React.FC = () => {
   return (
     <div>
-      <LoginScreen />
+      <BrowserRouter>
+        <Switch>
+          <Route path='/login' component={LoginScreen} />
+          <Route path='/blank-page' component={() => <h1>Página em branco</h1>} />
+          <Redirect to='/login' />
```

Comment:
Se você vai mandar sempre pro login por default, deixa a blankPage por primeiro, e faz o route `/` mandar pra login screen, aí você não precisa desse redirect.

OU você pode fazer uma rota de 404, pra caso entre um caso que a gente não espera, também é valio

```suggestion
          <Route path='/blank-page' component={() => <h1>Página em branco</h1>} />
          <Route path='/' component={LoginScreen} />
```